update word from currently to may (#2687)
facebook/react-native-website: docs/images.md

## iOS Border Radius Styles

> Please note that the following corner specific, border radius style properties are may ignored by iOS's image component:

Reviewer: ```suggestion
Please note that the following corner specific, border radius style properties might be ignored by iOS's image component:
```

Reviewer: I'm going to merge the suggestion and land the PR, so it won'y become stale. 😉 

Also the duplicates start to appear:
* https://github.com/facebook/react-native-website/pull/2908
